msteams-platform/graph-api/meeting-transcripts/get-transcripts-api.md

```diff
@@ -0,0 +1,880 @@
+```
+
+**Request headers**
+
+| Header       | Value |
+|:---------------|:--------|
+| Authorization  | Bearer {token}. Required.  |
+
+**Request body**
+
+Don't supply a request body for this method.
+
+**Response**
+
+If successful, this method returns a `200 OK` response code and a `callTranscript` object in the response body.
+
+<br>
+<details>
+<summary><b>Example: Get a callTranscript</b></summary>
+<br>
+<b>Request</b>
+<br>
+
+```http
+GET https://graph.microsoft.com/beta/users/ba321e0d-79ee-478d-8e28-85a19507f456/onlineMeetings/MSo1N2Y5ZGFjYy03MWJmLTQ3NDMtYjQxMy01M2EdFGkdRWHJlQ/transcripts/MSMjMCMjNzU3ODc2ZDYtOTcwMi00MDhkLWFkNDItOTE2ZDNmZjkwZGY4
+```
+
+<br>
+<b>Response</b>
+<br>
+
+> [!NOTE]
+> The response object shown here might be shortened for readability.
+
+```http
+HTTP/1.1 200 OK
+Content-type: application/json
+
+{
+    "@odata.context": "https://graph.microsoft.com/beta/$metadata#users('ba321e0d-79ee-478d-8e28-85a19507f456')/onlineMeetings('MSo1N2Y5ZGFjYy03MWJmLTQ3NDMtYjQxMy01M2EdFGkdRWHJlQ')/transcripts/$entity",
+    "id": "MSMjMCMjNzU3ODc2ZDYtOTcwMi00MDhkLWFkNDItOTE2ZDNmZjkwZGY4",
+    "createdDateTime": "2021-09-17T06:09:24.8968037Z"
+}
+```
+
+</details>
+
+### Get callTranscript content
+
+This API is used to get the transcript of the selected transcript ID that was obtained in the response of the `Get callTranscript` API. It returns the content of the transcript.
+
+**HTTP request**
+
+```http
+GET me/onlineMeetings({meetingId})/transcripts({transcriptId})/content
+GET users({userId})/onlineMeetings({meetingId})/transcripts({transcriptId})/content
+```
+
+**Optional query parameters**
+
+This method supports the `$format` [OData query parameter](/graph/query-parameters) that allows response customization.
+
+The supported format types are `text/vtt` for vtt OR `application/vnd.openxmlformats-officedocument.wordprocessingml.document` for docx format.
+
+**Request headers**
+
+| Header       | Value |
+|:---------------|:--------|
+| Authorization  | Bearer {token}. Required.  |
+| Accept  | text/vtt OR  application/vnd.openxmlformats-officedocument.wordprocessingml.document. Optional.  |
+
+**Request body**
+
+Don't supply a request body for this method.
+
+**Response**
+
+If successful, this method returns a `200 OK` response code and contains bytes for callTranscript object in the response body. `content-type` header specifies type of the transcript content.
+
+**Examples**
+<br>
+<details>
+<summary><b>Example 1: Get a callTranscript content</b></summary>
+<br>
+<b>Request</b>
+<br>
+
+```http
+GET https://graph.microsoft.com/beta/users/ba321e0d-79ee-478d-8e28-85a19507f456/onlineMeetings/MSo1N2Y5ZGFjYy03MWJmLTQ3NDMtYjQxMy01M2EdFGkdRWHJlQ/transcripts/MSMjMCMjNzU3ODc2ZDYtOTcwMi00MDhkLWFkNDItOTE2ZDNmZjkwZGY4/content
+```
+
+<br>
+<b>Response</b>
+<br>
+Response contains bytes for the transcript in the body. The `content-type` header specifies type of the transcript content.
```

Comment:
`content-type` isn't getting formatted correctly somehow in the rendered markdown

Reply:
Same comment for all the other examples below